CONTRIBUTING.md

```diff
@@ -48,6 +48,30 @@ All pull requests need to abide by the following criteria to be accepted:
 - support for localization in code (translations need not be provided)
 - tests for added / changed functionality
 
+## Working with Git
+
+It is a good idea to fork the `responsible-ai-toolbox` repo and work inside your own fork so that the main repo does not have to deal with so many branches and commits.
+
+### First Time Setup
+
+- Click on the `Fork` button in the top right corner of the [`responsible-ai-toolbox` repo.](https://github.com/microsoft/responsible-ai-toolbox) to create a fork.
+- Clone your forked repo to your local machine. Git will refer to your forked repo as `origin`.
+- Run `git remote add upstream https://github.com/microsoft/responsible-ai-toolbox.git` to register the upstream repo as `upstream`.
```

Comment:
you will probably want to use the ssh url instead of "https://github.com", so you won't need to type password in on syncs